pkg/logger/logger.go

```diff
@@ -32,7 +32,7 @@ var (
 	defaultOutput = os.Stderr
 )
 
-// Logger shared logger struct
+// Logger sharbeforeed logger struct
```

Comment:
:thinking:

Reply:
Same here copied `before` and pasted by mistake, sorry, I'll update it